packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -54,6 +59,7 @@ export const Combobox = <T extends string | number>({
   isClearable = false,
   createCustomValue = false,
   id,
+  inline = false,
```

Comment:
`inline`? maybe this should be `autoSize`?

However, I think I would just overload `width` prop for this.
 - `width={undefined}` is the default, full-width input
 - `width="auto"` is the auto-sizing input you're adding here
 - all other values are passed through and interpreted by `Input`

Thoughts?

Reply:
I really like this. Hopefully I'll be able to do some nice typing for it too.